### DISCUSSION TITLE
[Bug]: Masks/Zones editor weird behavior in Home Assistant

### DISCUSSION THREAD

**Discussion by bakedhorse** [blakeblackshear/frigate#11895] [Report a Bug] [answered]
### Describe the problem you are having

Using the Masks/Zones editor within Home Assistant doesn't behave like it should. It's harder to click on the points, and harder to create new points. Random spots on the image will be clickable and will move one of the points far away from where it actually is.

I found this to happen on the Home Assistant Frigate Proxy addon. Unsure if this occurs with the other Frigate Addons.

### Steps to reproduce

1. Setup frigate with Home Assistant
2. Attempt to edit masks/zones within the editor. It's almost impossible to create a new mask or zone.

### Version

0.14.0-272a21f

### Frigate config file

```yaml
Not relevant.
```

### Relevant log output

```shell
None.
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Mixed

### Camera make and model

Not relevant.

### Any other information that may be helpful

None.

**Comment by hawkeye217** [comment]
I just tested the mask editor through the HA proxy addon and I can't reproduce this at all, and we've had no other reports of this issue. What device and browser are you using?

**Reply by hawkeye217** [reply, marked as answer]
Some of the privacy features in Brave are preventing the javascript library Frigate uses for canvas drawing (Konva) from working correctly.

https://github.com/konvajs/konva/issues/1132#issuecomment-867339732

> Client-side fix in Brave: disable its "Brave Shields".

**Reply by bakedhorse** [reply]
> Some of the privacy features in Brave are preventing the javascript library Frigate uses for canvas drawing (Konva) from working correctly.
> 
> [konvajs/konva#1132 (comment)](https://github.com/konvajs/konva/issues/1132#issuecomment-867339732)
> 
> > Client-side fix in Brave: disable its "Brave Shields".

@hawkeye217  Ahh, thank you. Thought it was odd considering it worked fine when directly talking to frigate's UI on port 5000.
